docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -279,6 +204,85 @@ spec:
             name: worker-user-data
 ```
 
+Once the workers are deployed, you must label them SR-IOV capable:
```

Comment:
Not sure if this is more correct but it sounds better to my ears as a non-native English speaker: 
```suggestion
Once the workers are deployed, you must label them as SR-IOV capable:
```